lib/hexpm/billing/billing.ex

```diff
@@ -35,4 +35,15 @@ defmodule Hexpm.Billing do
         {:error, reason}
     end
   end
+
+  def update(organization, params, audit: audit) do
+    case impl().update(organization, params) do
+      {:ok, result} ->
+        Repo.insert!(audit(audit.audit_data, "billing.update", {audit.organization, params}))
```

Comment:
I need to use `audit.organization` here because `organization` is already used.

But in `create/2`, we use `organization` instead of `audit.organization`.

Do we need to make them consistent by changing `create/2`?

Reply:
How about renaming the first argument to `organization_name` instead?

Reply:
Sure. It's updated now, please check